src/app/routes/radioPodcast/getInitialData/index.js

```diff
@@ -66,7 +76,7 @@ export default async ({ path: pathname, pageType, service, toggles }) => {
     return {
       status,
       pageData: {
-        metadata: { type: 'On Demand Radio' },
+        metadata: { type: detailPageType },
```

Comment:
We need to pass through brandId too, required for recentEpisodes links

https://github.com/bbc/simorgh/pull/8766/files#diff-7a0fe662d3653d7e9cea8671ad812edc4182cc100b1e31fd03c20361b783c8ecR85